actions_helper/commands/create_task_definition.py

```diff
@@ -0,0 +1,89 @@
+from typing import Any
+
+from botocore.client import BaseClient
+
+from actions_helper.commands.get_active_task_definition_by_tag import get_active_task_definition_arn_by_tag
+from actions_helper.outputs import CreateTaskDefinitionOutput
+from actions_helper.utils import set_error
+
+PLACEHOLDER_TEXT = "PLACEHOLDER"
+
+
+def describe_task_definition(ecs_client: BaseClient, task_definition_arn: str, image_uri: str):
+    task_definition = ecs_client.describe_task_definition(taskDefinition=task_definition_arn)["taskDefinition"]
+
+    all_container_images = [
+        container_definition["image"] for container_definition in task_definition["containerDefinitions"]
+    ]
+
+    if len(set(all_container_images)) != 1 or all_container_images[0] != PLACEHOLDER_TEXT:
+        set_error("Not all values for containerDefinitions 'image' equal to 'PLACEHOLDER'")
```

Comment:
Will this simplify the logic?

```suggestion
    try:
        (container_image,) = {
            container_definition["image"] for container_definition in task_definition["containerDefinitions"]
        }
        if container_image != PLACEHOLDER_TEXT:
            set_error(f"Not all values for containerDefinitions 'image' equal to '{PLACEHOLDER_TEXT}'")
    except ValueError:
        set_error("Only a single image name is allowed")
```